astropy/io/misc/hdf5.py

```diff
@@ -238,6 +238,7 @@ def write_table_hdf5(table, output, path=None, compression=False,
     path : str
         The path to which to write the table inside the HDF5 file.
         This should be relative to the input file or group.
+        If not specified, defaults to __astropy__
```

Comment:
Nitpick: Just to make things clearer in rendered doc?
```suggestion
        If not specified, defaults to ``__astropy__``.
```